hub-templates/base-hub/templates/docs-service-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{ if .Values.docsService.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: docs-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: docs-service
+  template:
+    metadata:
+      labels:
+        app: docs-service
+    spec:
+      volumes:
+        - name: nginx-config
+          configMap:
+            name: nginx-docs-configmap
+        - name: docs
+          emptyDir: {}
+      initContainers:
+        - name: docs-clone
+          image: alpine/git
+          args:
+            - clone
+            - --depth=1
+            - -b
+            - gh-pages
```

Comment:
The branch should be configurable too, since many repos just use master instead of gh-pages.